internal/export/distro/filter_test.go

```diff
@@ -22,10 +23,13 @@ const (
 	descriptor2 = "Descriptor2"
 )
 
-func TestFilterDevice(t *testing.T) {
-	logger = zap.NewNop()
-	defer logger.Sync()
+func init() {
+	if LoggingClient == nil {
+		LoggingClient = logger.NewClient(internal.ExportDistroServiceKey, false, "")
+	}
+}
```

Comment:
My preference would be to setup these kinds of dependencies in a TestMain function. See internal/core/data/event_test.go